nengo_bones/templates/remote.sh.template

```diff
@@ -112,6 +134,20 @@ See https://docs.travis-ci.com/user/encrypting-files/ for more details.
             echo "({{ host }}) Failed to create conda environment. Exiting.";
             exit \$REMOTE_STATUS;
         fi
+
+        {% if conda_install %}
+        # Install necessary conda packages
+        echo "$ ({{ host }}) Conda installing {%- for conda_pkg in conda_install %} "{{ conda_pkg }}" {%- endfor %}"
+        {% for conda_pkg in conda_install %}
```

Comment:
I think this functionality is already supported by the `remote_setup` option in the remote script. E.g., see https://github.com/nengo/nengo-dl/blob/master/.nengobones.yml#L202

Reply:
Ohhh. This should be documented. 😆

Reply:
Looking into this further, it looks like `remote_setup` is only available if you use the `remote-script` template, which not all repos do, because the actual script itself is highly variable. Putting the `conda install` here would make it more general (or, I could also move the `remote_setup` block into the `remote` template.

Reply:
If you're using the `remote.sh` script directly then you have full control over the `remote_script` block, which you could use to do the conda installation. `remote_setup` is only needed if you're using `remote-script.sh`, as then it lets you insert functionality within the `remote_script` block (since `remote-script.sh` is "taking control" of that block).